Automattic/jetpack
# Add tracking to Jetpack review link on plugins page

Related https://github.com/Automattic/jetpack/pull/34998

## Proposed changes:
We direct the review link through Jetpack Redirect so as to track clicks on that link.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


We will track clicks on "5-star link" under the plugin listed on installed plugins page.

## Testing instructions:
* Activate the branch on Jetpack plugin through Jetpack Beta on e.g. Jurassic Ninja
* Connect user to Jetpack (necessary from a technical standpoint, but it makes sense - users need to use Jetpack to give valid reviews!)
* Go to `/wp-admin/plugins.php` (Installed plugins page)
* Click on "5 stars":
![CleanShot 2024-03-12 at 15 40 36@2x](https://github.com/Automattic/jetpack/assets/8419292/f9f046bc-7994-4dee-91ae-1d54bd927255)

* Go to Tracks MC `/live/?eventname=jetpack_plugin_redirect&user=<your_username>` (only a12s) and after ~5 mins check if the valid Tracks event appeared
